packages/teams-js/test/private/teams.spec.ts

```diff
@@ -250,7 +250,6 @@ describe('Testing teams capabillity', () => {
     });
 
     describe('joinedTeams.isSupported function', () => {
-      const utils = new Utils();
```

Comment:
Why was this removed?

Reply:
We are setting const utils = new Utils() above in the beforeEach(), it seems unnecessary to do it at the beginning of this describe when it's already being reset.